coalib/bearlib/abstractions/Linter.py

```diff
@@ -612,9 +612,9 @@ def linter(executable: str,
         A custom command to check for when ``check_prerequisites`` gets
         invoked (via ``subprocess.check_call()``). Must be an ``Iterable``.
     :param prerequisite_check_fail_message:
-        A custom command to check for when ``check_prerequisites`` gets
-        invoked. Must be provided only together with
-        ``prerequisite_check_command``.
+        A custom message that gets displayed when ``check_prerequisites``
+        fails when invoking ``prerequisite_check_command``. Can only be
```

Comment:
could you use while instead of when? it hurts my psychically having when x when y; too many ``when``s